OWNERS

```diff
@@ -1,15 +1,13 @@
 reviewers:
+  - alaypatel07
```

Comment:
you don't want him here.  He won't be actively reviewing, so it can skew assignments.